app/Main.hs

```diff
@@ -41,6 +34,8 @@ optionsParser = KrankOpts
        <$> githubKeyToParse
        <*> (Opt.switch $ Opt.long "dry-run"
         <> Opt.help "Perform a dry run. Parse file, but do not execute HTTP requests")
+      <*> (not <$> (Opt.switch $ Opt.long "no-colors"
+           <> Opt.help "Disable colored outputs"))
```

Comment:
nit: I have a hard time matching the blocks. Can we improve the indentation/formatting and/or extract a piece of the code?

Reply:
like I have extracted each option in its own parser. Makes it more readable although a bit more verbose.

Reply:
Good point.